packages/datetime/src/common/classes.ts

```diff
@@ -36,6 +36,7 @@ export const DATEPICKER_FOOTER = `${DATEPICKER}-footer`;
 export const DATEPICKER_MONTH_SELECT = `${DATEPICKER}-month-select`;
 export const DATEPICKER_YEAR_SELECT = `${DATEPICKER}-year-select`;
 export const DATEPICKER_NAVBAR = `${DATEPICKER}-navbar`;
+export const DATEPICKER_NAVBUTTON = `${DATEPICKER}-NavButton`;
```

Comment:
actually it's 
```suggestion
export const DATEPICKER_NAVBUTTON = `DayPicker-NavButton`;
```

sorry, I know the "date" vs "day" thing is confusing in this file